bundle-workflow/README.md

```diff
@@ -307,7 +308,28 @@ Check for updates and create any new manifests.
 
 The following options are available.
 
-| name               | description                                                             |
-|--------------------|-------------------------------------------------------------------------|
-| --keep             | Do not delete the temporary working directory on both success or error. |
-| -v, --verbose      | Show more verbose output.                                               |
+| name          | description                                                             |
+| ------------- | ----------------------------------------------------------------------- |
+| --keep        | Do not delete the temporary working directory on both success or error. |
+| -v, --verbose | Show more verbose output.                                               |
+
+### Component Onboarding
```

Comment:
Looks like we are duplicating information. 
Most of this is already captured in https://github.com/opensearch-project/opensearch-build/blob/main/ONBOARDING.md. 

If it is for component owners, I would rather let it go in ONBOARDING.md as a single place to look for and this README.md can link to it if needed.